pyproject.toml

```diff
@@ -26,7 +26,7 @@ docs = [
 ]
 
 [project.urls]
-repository = "magiepy.readthedocs.io"
+repository = "https://magie.ie"
```

Comment:
I suppose `https://magiepy.readthedocs.io/` would be fine too, but if nothing is there?